examples/tutorials/lighting/app.ts

```diff
@@ -36,6 +37,10 @@ void main(void) {
   vPosition = (app.modelMatrix * vec4(positions, 1.0)).xyz;
   vNormal = mat3(app.modelMatrix) * normals;
   vUV = texCoords;
+
+  #if (defined(LIGHTING_VERTEX))
+  vColor = lighting_getLightColor(vec3(1.0), app.eyePosition, vPosition, normalize(vNormal));
```

Comment:
Thought: Maybe we could name the vertex and fragment calculation functions differently and export a trivial stub in one of the modules, so that we can avoid #ifdefs which we don't have naturally in WGSL.

This will come into focus on master where we also need to deal with the WGSL implementation.